smtk-import/CMakeLists.txt

```diff
@@ -35,6 +35,7 @@ pkg_config_library(LIBGIT2 libgit2 REQUIRED)
 pkg_config_library(LIBMDB libmdb REQUIRED)
 pkg_config_library(LIBDC libdivecomputer REQUIRED)
 pkg_config_library(LIBFTDI libftdi1 QUIET)
+pkg_config_library(LIBRAW libraw)
```

Comment:
Is this actually needed here - not sure if smtk-import is handling images anywhere?

Reply:
the link step fails without it. So a component that is linked into smtk2ssrf requires that library.
We can argue that that shouldn't be the case - but my solution above at least solves the failing builds.

Reply:
@dirkhh: I figured out the 'why' for this - our CICD builds for smtk2ssrf are re-using the compiler output from the subsurface build that they do beforehand - and this build has got LIBRAW configured.

Do you know what the purpose is for this re-use?

Reply:
I found something in the way of an explanation:

https://github.com/subsurface/subsurface/blob/a6a15f9d3a75262220c1d3e6f8f8e20759b9061a/packaging/windows/smtk2ssrf-mxe-build.sh#L8-L12

I am not sure if this re-use of the library that was built for Subsurface makes sense in the way we are delivering the apps at the moment:
- we are dragging the dependencies that Subsurface requires into smtk2ssrf, if they are needed or not;
- we seem to be only delivering smtk2ssrf with its own installer, and its own packaged `libsubsurface_corelib`.

When I looked around, I could only find the standalone Windows installer as a user facing distribution format for smtk2ssrf - do you know if smtk2ssrf is included in any of the packages we are building for linux distributions?

Reply:
This was wrote thinking in a fresh build from source in a local machine, as build.sh ensures everything is in place. Then smtk2ssrf-build.sh rebuilds libsubsurface_corelib.a dropping some unused dependencies like USB or BT.  If libraw is not built into corelib it should be possible to drop it too from smtk2ssrf builds.

Reply:
> When I looked around, I could only find the standalone Windows installer as a user facing distribution format for smtk2ssrf - do you know if smtk2ssrf is included in any of the packages we are building for linux distributions?

No AFAIK. It's an standalone app and is not supposed to be shipped with Subsurface. BTW is possible to build an CLI version which is used by @atdotde in his web page.